Sources/xcodeproj/Objects/Project/PBXProjEncoder.swift

```diff
@@ -148,55 +212,96 @@ final class PBXProjEncoder {
         self.multiline = beforeMultiline
         writeNewLine()
     }
-
+    
     private func writeDictionaryStart() {
         output.append("{")
         if multiline { writeNewLine() }
         increaseIndent()
     }
-
+    
     private func writeDictionaryEnd() {
         decreaseIndent()
         writeIndent()
         output.append("}")
     }
-
+    
     private func write(array: [PlistValue]) {
         writeArrayStart()
         array.forEach { write(arrayValue: $0) }
         writeArrayEnd()
     }
-
+    
     private func write(arrayValue: PlistValue) {
         writeIndent()
         write(value: arrayValue)
         output.append(",")
         writeNewLine()
     }
-
+    
     private func writeArrayStart() {
         output.append("(")
         if multiline { writeNewLine() }
         increaseIndent()
     }
-
+    
     private func writeArrayEnd() {
         decreaseIndent()
         writeIndent()
         output.append(")")
     }
-
+    
     private func writeIndent() {
         if multiline {
             output.append(String(repeating: "\t", count: Int(indent)))
         }
     }
-
+    
     private func increaseIndent() {
         indent += 1
     }
-
+    
     private func decreaseIndent() {
         indent -= 1
     }
+    
+    private func sortFiles<T>(_ obj: T) where T: PBXObject {
```

Comment:
Nitpick: I see too many empty lines here. Would you mind running `swift format .`?

Reply:
Which formatter are you talking about. On my system there isn't any `swift format` command available. I've used https://github.com/nicklockwood/SwiftFormat in the past.